src/uiveri5.js

```diff
@@ -318,6 +318,25 @@ function run(config) {
         });
       };
 
+      browser.get = function (sUrl, iTimeout) {
```

Comment:
Overriding the browser.get is a change in our current "strategy" where we have browser.testrunner.navigate API, modeled according to browser.driver.navigate API.
I am generally fine but my concern is how to fit the declarative authentication and how to cope with the implicit expectation that every test should start with browser.get